src/components/ui/searchBar.tsx

```diff
@@ -0,0 +1,37 @@
+import React from "react";
+import { Input } from "./input";
+import { SearchIcon } from "./searchIcon";
+
+interface Props {
+  placeholder?: string;
+  value?: string;
+  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onClick?: () => void;
+}
+
+export default function SearchBar({
```

Comment:
Can't that be done with = React.forwardRef<HTMLInputElement, InputProps> element? Same as all the others UI components?

Or at least = React.forwardRef<
  HTMLDivElement,
  React.HTMLAttributes<HTMLDivElement>
> if it is essential to be div instead of input

Reply:
no